sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusSessionReceiverClient.java

```diff
@@ -13,14 +15,15 @@
 /**
  * This session receiver client is used to acquire session locks from a queue or topic and create
  * {@link ServiceBusReceiverClient} instances that are tied to the locked sessions.
- * Use {@link #acceptSession(String)} to acquire the lock of a session if you know the session id.
- *
- * {@codesnippet com.azure.messaging.servicebus.servicebusreceiverclient.instantiation#nextsession}
- *
- * Use {@link #acceptNextSession()} to acquire the lock of the next available session without specifying the session id.
  *
+ * <p><strong>Receive messages from a specific session</strong></p>
+ * <p>Use {@link #acceptSession(String)} to acquire the lock of a session if you know the session id.</p>
```

Comment:
When user create  this  client `ServiceBusSessionReceiverClient`, they have to close this. Because this is intermediate client, it is easy to miss. We had same issue in our live test.
So may be we can write this  in builder when this is created also here so User ensure this is closed.

Reply:
The code snippets have this well documented.